kolibri/core/templates/kolibri/base.html

```diff
@@ -1,6 +1,7 @@
 {% load i18n kolibri_tags webpack_tags js_reverse cache %}<html>
 <head>
   <title>{% block title %}{% endblock %} - {% trans "Kolibri" %}</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
```

Comment:
@rtibbles is it ok that i added this meta tag in the base.html file, or should this tuck away somewhere else?